auth4genai/snippets/mcp/get-started/pre-reqs/enable-resource-param-step.mdx

```diff
@@ -4,19 +4,4 @@ The quickest way to enable it is through the [Auth0 Dashboard](https://manage.au
 
 1. Navigate to **Settings** on the left sidebar.
 2. Click on [**Advanced**](https://manage.auth0.com/dashboard/#/tenant/advanced) on the top right corner.
-3. Scroll down to the Settings section, find and enable **Resource Parameter Compatibility Profile**.
-
-  <Frame>
-    <img
-      className="hidden dark:block"
-      src="/img/mcp/enable_resource_param_compatibility_dark.png"
-      alt="Resource Parameter Compatibility Profile enabled"
-    />
-    <img
-      className="block dark:hidden"
-      src="/img/mcp/enable_resource_param_compatibility_light.png"
-      alt="Resource Parameter Compatibility Profile enabled"
-    />
-  </Frame>
-
-Resource Parameter Compatibility Profile is available under the Early Access program. Go to the [Early Access section](/mcp/intro/overview#early-access) to learn how to gain access.
+3. Scroll down to the Settings section, find and enable the **Resource Parameter Compatibility Profile** toggle.
```

Comment:
might be worth calling out this is part of EA program? and link out to the [EA section](https://auth0.com/ai/docs/mcp/intro/overview#early-access)?